src/can.c

```diff
@@ -627,7 +778,7 @@ static void thingset_can_thread()
     }
 
     while (true) {
-        thingset_can_process_inst(&ts_can_single, K_FOREVER);
+        k_sleep(K_FOREVER);
```

Comment:
Should the `process` function not be kept if `CONFIG_ISOTP_FAST` is not used?

Also when it's used we don't need the overhead and additional stack of the thread anymore, if it sits there and sleeps forever. We should use `SYS_INIT` instead.